src/resources/postcss/components/_messages.pcss

```diff
@@ -0,0 +1,52 @@
+.tribe-events {
+	/* -----------------------------------------------------------------------------
+	 *
+	 * Messages
+	 *
+	 * Example:
+	 * <div class="tribe-events-c-messages">
+	 *   <div class="tribe-events-c-messages__inner">
+	 *     <span>There were no results found</span>
+	 *   </div>
+	 * </div>
+	 *
+	 * ----------------------------------------------------------------------------- */
+
+	.tribe-events-c-messages {
+		background-color: var(--color-background-messages);
+		border-radius: var(--border-radius-default);
+		padding: var(--spacer-2);
+		margin: 0 0 var(--spacer-3) auto;
+		width: 100%;
+		flex: 0 0 100%;
+		order: -1;
+
+		@media (--viewport-medium) {
+			order: 3;
+			padding: var(--spacer-3);
+			margin: 0 0 var(--spacer-7) auto;
+		}
+	}
+
+	.tribe-events-c-messages__inner {
```

Comment:
do we know what this will look like with multiple messages? do we need spacing below each message?

Reply:
No, so far the designs are for one message only. But the data structure allows more than one. That's why I did it this way.

Reply:
If we're going to be using lists and foreach loops, we should be accounting for this in the styles as well.